modules/apps/frontend-theme/frontend-theme-classic/src/css/_custom.scss

```diff
@@ -102,3 +102,7 @@ body {
 		padding-top: 3rem;
 	}
 }
+
+.media:has(div[data-oembed-url]) {
+	display: inherit;
+}
```

Comment:
Media plugin is very opinionated in HTML it produces, [see here](https://ckeditor.com/docs/ckeditor5/latest/features/media-embed.html#previewable-media). It clashes with Clay's `.media` style, so we need to override it.